packages/gatsby-plugin-sharp/src/gatsby-node.js

```diff
@@ -1,6 +1,18 @@
-const { setBoundActionCreators, setPluginOptions } = require(`./index`)
+exports.onPreInit = ({ actions, reporter }, pluginOptions) => {
+  try {
+    require(`sharp`)
+  } catch (error) {
+    // Bail early if sharp isn't working
+    // TODO: Add link to docs
+    reporter.panic(
+      `
+      "sharp" doesn't seem to have been built or installed correctly
```

Comment:
Can we add some additional info here?

This doesn't seem like it'd be any _more_ useful to a user as they go through debugging why this error is triggered?

e.g. 

```js
reporter.panic(
  reporter.stripIndent`
    The dependency "sharp" does not seem to have been built or installed correctly.
    Check out this guide for more info: https://gatsby.app/sharp-installation-issues
  `
)
```

Reply:
(also note the link doesn't--and probably shouldn't--be something we maintain)